test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md

```diff
@@ -48,7 +48,7 @@ This sequence can be run as follows
 ./network.sh deployCCAAS  -ccn basicts -ccp ../asset-transfer-basic/chaincode-typescript
 ```
 
-This is very similar to the `deployCC` command, it needs the name, and path. But also needs to have the port the chaincode container is going use. As each container is on the `fabric-test` network, you might wish to alter this so there are no collisions with other chaincode containers.
+This is very similar to the `deployCC` command, it needs the name, and path. But also needs to have the port the chaincode container is going to use. As each container is on the `fabric-test` network, you might wish to alter this so there are no collisions with other chaincode containers.
```

Comment:
I would be a bit more clear on this. Just say `it is recommended that you should not use the chaincode name basic because it will collide with other chaincodes that you may have deployed on the test network`